ssn/chapters/Common.html

```diff
@@ -325,36 +315,34 @@ <h6><a href="#SOSAisFeatureOfInterestOf">sosa:isFeatureOfInterestOf</a></h6>
             </p>
             <p>
                 <em>is feature of interest of</em> &mdash;
-                A relation between a <a href="#SOSAFeatureOfInterest">FeatureOfInterest</a> and an
-                <a href="#SOSAExecution">Execution</a> or <a href="#SOSADeployment">Deployment</a> operating on it,
+                relation between a FeatureOfInterest and an
```

Comment:
Another article dropped from the sentence. Looks intentional, just the heads up.

Reply:
Looks unintentional to me.
```suggestion
                A relation between a FeatureOfInterest and an
```

Reply:
See response above https://github.com/w3c/sdw-sosa-ssn/pull/340#discussion_r2199391437